index.js

```diff
@@ -1,18 +1,26 @@
 module.exports = () => {
+  function processSize (propPrefix, decl, list) {
```

Comment:
Move this function to the root level to avoid re-definition on every plugin call.